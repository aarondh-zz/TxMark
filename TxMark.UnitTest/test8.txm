<div>
  This should be "three": (print:$array's 3rd)
  This should also be "three": (print:$array's (3))
  This should also be "three": (print: 3rd of $array)
  This should also be "three": (print: (5 * 2 - 7) of $array)
  This should be "6": (print: length of $array)
</div>
