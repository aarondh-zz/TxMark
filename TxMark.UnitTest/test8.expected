<div>
This should be &quot;three&quot;: three
This should also be &quot;three&quot;: three
</div>
